copier.yml

```diff
@@ -1,16 +1,8 @@
 # Configure jinja2 defaults to make syntax highlighters lives easier
 _templates_suffix: .jinja
-_envops:
```

Comment:
Why do you remove this?

Reply:
see https://github.com/OCA/oca-addons-repo-template/pull/221#discussion_r1377253209